requirements.yaml

```diff
@@ -0,0 +1,9 @@
+name: test
```

Comment:
This file is commonly named `environment.yaml`, shall we use that here too?

Maybe name this conda environment as `s1reader` to be distinguishable with future tests for other packages like `compass`?